parlai/core/params.py

```diff
@@ -1078,10 +1087,10 @@ def _kwargs_to_str_args(self, **kwargs):
                 string_args.append(last_option_string)
             elif isinstance(action, argparse._StoreAction) and action.nargs is None:
                 string_args.append(last_option_string)
-                string_args.append(str(value))
+                string_args.append(self._value2argstr(value))
             elif isinstance(action, argparse._StoreAction) and action.nargs in '*+':
                 string_args.append(last_option_string)
-                string_args.extend([str(v) for v in value])
+                string_args.extend([self._value2argstr(value) for v in value])
```

Comment:
lol okay so THIS one is needs to be space separated, while the one  above needs to be comma separated.

sorry, nothing's easy.